### PR TITLE
Workaround for accessing the webhook's query in the body of the Webhook

### PR DESCRIPTION
**Pull request by zubairov**
Related to https://github.com/elasticio/webhooks/issues/70

### CODE REVIEW

File: receive.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ exports.process = function (msg, conf) {
     console.log("Received new message with id", msgId);
     console.log(msg);
 
+    if (msg.query && msg.body) {
```
Comment (jhorbulyk) [inline receive.js]: What if we have query string params on a GET request?

Comment (zubairov) [reply receive.js]: @jhorbulyk you mean something like:

```
https://in.elastic.io/hooks/123456789?query=foo
```
?

Comment (jhorbulyk) [reply receive.js]: Correct.

Comment (drobiazko) [reply receive.js]: they result in `msg.body` anyway

Comment (jhorbulyk) [reply receive.js]: When would `body` not be defined?  If one does a POST or similar with no body, is there something to attach the query string params to?

Comment (drobiazko) [reply receive.js]: `msg.body` is always defined.

**Comment by zubairov** [comment]
@luzhok @stas-fomenko how can I facilitate the test of this fix?

**Comment by luzhok** [comment]
@zubairov where can we test it? is it already on popeye or sparrow?

**Comment by zubairov** [comment]
@luzhok tbh no idea, code is merged into the components `master` branch what is the usual process after that?

Expected behavior after this fix would be following:
- If webhook is submitted as before (e.g. POST with json or any other content types) no changes in behaviour is expected
- if webhook is submitted as before _but_ webhook URL has additional GET parameters in it (e.g. _https://in.elastic.io/hoooks/123456798<b>?foo=baz</b>_ ) then additional field called `_query` will appear inside the body of the message after the `WebHook` component and values in it are parsed from query parameters (e.g. in the sample above ``{"foo":"baz"}``)

**Comment by zubairov** [comment]
@khanzadyan may be we need to mention it in the documentation too (once it will be merged and deployed to production)

**Comment by luzhok** [comment]
@zubairov ok, I see ) I'll push the component to sparrow and test it. I'll inform you about the test results

**Comment by zubairov** [comment]
@khanzadyan now it's a good time to mention it in the documentation, I'm ready to make a suggestion (PR) but need your help to show me where?

**Comment by khanzadyan** [comment]
@zubairov can you please create an issue in here: https://github.com/elasticio/elasticio.github.io/issues

**Comment by zubairov** [comment]
@khanzadyan done - https://github.com/elasticio/elasticio.github.io/issues/26

**Pull request merged**